tests/devapps/NetCoreTestApp/Program.cs

```diff
@@ -75,6 +75,7 @@ private static string GetAuthority()
 
         private static IPublicClientApplication CreatePca(bool withWamBroker = false)
         {
+            // <PCABootstrapSample>
```

Comment:
What is this for? Note that this isn't great as a sample, because the token caching is done in a plaintext file.